servers/node_server.js

```diff
@@ -1,12 +1,14 @@
 const cluster = require('cluster');
 const http = require('http');
 const numCPUs = require('os').cpus().length;
-
+const threads = process.argv.length>2?parseInt(process.argv[2]):numCPUs;
```

Comment:
Again passing number of processes (not threads) as an argument is not a prerequisite. 
Also in this case (on my workstation), augmenting them does not offer any benefit.